README.md

```diff
@@ -249,8 +249,8 @@ async def quote_callback(q):
 # Initiate Class Instance
 stream = Stream(<ALPACA_API_KEY>,
                 <ALPACA_SECRET_KEY>,
-                base_url=URL('https://paper-api.alpaca.markets'),
-                data_feed='iex')  # <- replace to SIP if you have PRO subscription
+                base_url='https://paper-api.alpaca.markets',
+                data_feed='iex')  # <- replace to 'sip' if you have PRO subscription
```

Comment:
Can you please add this comment to the second example below as well please?

Reply:
Sure @drew887 just added it.